README.md

```diff
@@ -50,787 +50,6 @@ You will need the following prerequisites:
   - Make sure your API key has the
     following [capabilities](https://ably.com/docs/auth/capabilities): `publish`, `subscribe`, `presence`, `history` and `channel-metadata`.
 
-## Installation
-
-Install the Chat SDK:
-
-```sh
-npm install @ably/chat
-```
-
-For browsers, you can also include the Chat SDK directly into your HTML:
-
-```html
-<!-- Ably Chat also requires the core Ably SDK to be available -->
-<script src="https://cdn.ably.com/lib/ably.min-2.js"></script>
-<script src="https://cdn.ably.com/lib/ably-chat.umd.cjs-0.js"></script>
-<script>
-  const realtime = new Ably.Realtime({ key: 'your-ably-key' });
-  const chatClient = new AblyChat.ChatClient(realtime);
-</script>
-```
-
-> [!IMPORTANT]
-> If you're using Chat in React Native, please see the additional setup step [here](./src/react/README.md).
-
-## Versioning
-
-The Ably client library follows [Semantic Versioning](http://semver.org/). To lock into a major or minor version of the client library, you can specify a specific version number such as https://cdn.ably.com/lib/ably-chat-0.js for all v0._ versions, or https://cdn.ably.com/lib/ably-chat-0.1.js for all v0.1._ versions, or you can lock into a single release with https://cdn.ably.com/lib/ably-chat-0.1.0.js. See https://github.com/ably/ably-chat-js/tags for a list of tagged releases.
```

Comment:
Maybe leave this in? Remove the react native bit, since we're merging packages soon we won't need it anyway.

Reply:
Yeah that's reasonable :)

Reply:
Done!